ocs_ci/utility/utils.py

```
@@ -2292,6 +2297,27 @@ def stringify(value):
         all_args_string = ", ".join(args + kwargs)
         return f"{self.func.__name__}({all_args_string})"
 
+    def _log_progress(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_log_progress`

Add return type annotation: `None`

(ANN202)